drydock_provisioner/control/middleware.py

```diff
@@ -66,7 +66,7 @@ def process_request(self, req, resp):
 
         requested_logging = req.get_header('X-Log-Level')
 
-        if (config.DrydockConfig.global_config.get('log_level', '') == 'DEBUG' or
+        if (drydock_provisioner.conf.logging.log_level == 'DEBUG' or
```

Comment:
Updated to use cfg.CONF, though did not create a module variable CONF. Just reference cfg.CONF throughout.